edk2toolext/edk2_logging.py

```diff
@@ -255,6 +255,13 @@ def __init__(self):
         self._verbose = False
         self._currentSection = "root"
 
+        secrets_regex_strings = [
+            r"[A-Za-z0-9]{46}",  # Nuget API Key
+            r"github_pat_[^ ]*"  # Github PAT
```

Comment:
The GitHub PATs I've seen of the format `ghp_<value>`.

Like what is described here: [Authentication token format updates are generally available](https://github.blog/changelog/2021-03-31-authentication-token-format-updates-are-generally-available/).

The prefix changes between token types. I think we should cover all defined types.

This won't cover the previous token format. I think that's fine given its ambiguity (that led GitHub to this change) and the fact the new format has been in effect for nearly two years now.

Reply:
Thanks for sending me that documentation! I guess what I found was old. I've Updated to support gh[pousr]_.